cmd/nginx-ingress/flags.go

```diff
@@ -135,6 +135,15 @@ var (
 	prometheusMetricsListenPort = flag.Int("prometheus-metrics-listen-port", 9113,
 		"Set the port where the Prometheus metrics are exposed. [1024 - 65535]")
 
+	enableHealthProbe = flag.Bool("enable-health-probe", false,
+		`Enable health probe for external load balancers. Requires -nginx-plus`)
+
+	healthProbeTLSSecretName = flag.String("health-probe-tls-secret", "",
+		`A Secret with a TLS certificate and key for TLS termination of the health probe.`)
+
+	healthProbeListenPort = flag.Int("health-probe-listen-port", 9000,
+		"Enable health probe and set the port where the health probe endpoint are exposed. Requires -nginx-plus. [1024 - 65535]")
+
```

Comment:
Our docs will need to be updated with these new CLI arguments.
The file you need to update is `docs/content/configuration/global-configuration/command-line-arguments.md`